src/EmployerPR/SFA.DAS.EmployerPR/InnerApi/Requests/GetAccountLegalEntitiesRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerPR.InnerApi.Requests;
+public class GetAccountLegalEntitiesRequest : IGetAllApiRequest
+{
+    private readonly string _hashedAccountId;
+
+    public GetAccountLegalEntitiesRequest(string hashedAccountId)
+    {
+        _hashedAccountId = hashedAccountId;
+    }
+
+    public string GetAllUrl => $"api/accounts/{_hashedAccountId}/legalentities?includeDetails=true";
```

Comment:
Do we not have a request to this endpoint in shared library? if not then we should add this to shared lib